community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf

```diff
@@ -470,6 +470,16 @@ variable "enable_default_mounts" {
   default     = true
 }
 
+variable "enable_controller_default_mounts" {
+  description = <<-EOD
+    Enable default global network storage to controller.
```

Comment:
does this get disabled if a /home is passed via `use`? if so, you might want to add it to the description.